helper/staking/staking.go

```diff
@@ -110,18 +115,22 @@ var (
 )
 
 // PredeployStakingSC is a helper method for setting up the staking smart contract account,
 // using the passed in validators as prestaked validators
 func PredeployStakingSC(
 	validators []types.Address,
+	minValidatorCount uint,
+	maxValidatorCount uint,
```

Comment:
What do you think about extracting these params into a structure like `PredeployParams`?

Reply:
I agree 👍 Added in https://github.com/0xPolygon/polygon-edge/pull/397/commits/fe34bdd49d2f6aae469ffc12ae67d33f89fe280f